File: Exercicios-Beecrowd/C#/1012.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Teste
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] lista= Console.ReadLine().Split();
            double a = Convert.ToDouble(lista[0]);
            double b = Convert.ToDouble(lista[1]);
            double c = Convert.ToDouble(lista[2]);
            double triangulo = (a * c) / 2;
            double circulo = Math.Pow(c, 2)* 3.14159 ;
            double trapezio = ((a + b) * c) / 2;
            double quadrado = b* b;
            double retangulo = a * b;
            string tri = triangulo.ToString("F3");
            string cir = circulo.ToString("F3");
            string tra = trapezio.ToString("F3");
            string qua = quadrado.ToString("F3");
            string ret = retangulo.ToString("F3");
            Console.WriteLine("TRIANGULO: {0}",tri);
            Console.WriteLine("CIRCULO: {0}", cir);
            Console.WriteLine("TRAPEZIO: {0}", tra);
            Console.WriteLine("QUADRADO: {0}", qua);
            Console.WriteLine("RETANGULO: {0}", ret);


        }
    }
}

File: Exercicios-Beecrowd/Kotlin/1012.kt
import java.util.*

fun main(args: Array<String>) {

    val valor = readLine().toString().split(" ")
    val a=valor[0].toDouble()
    val b=valor[1].toDouble()
    val c=valor[2].toDouble()
    val triangulo=(a*c)/2
    val circulo=(c*c)*3.14159
    val trapezio=(a+b)*c/2
    val quadrado=b*b
    val retangulo=a*b
    println("TRIANGULO: ${"%.3f".format(triangulo)}")
    println("CIRCULO: ${"%.3f".format(circulo)}")
    println("TRAPEZIO: ${"%.3f".format(trapezio)}")
    println("QUADRADO: ${"%.3f".format(quadrado)}")
    println("RETANGULO: ${"%.3f".format(retangulo)}")
	
}